pkg/scyllaclient/client_scylla_integration_test.go

```diff
@@ -129,7 +130,12 @@ func TestClientDescribeRingIntegration(t *testing.T) {
 	for i := range testCases {
 		tc := testCases[i]
 		t.Run(tc.name, func(t *testing.T) {
-			if err := clusterSession.ExecStmt(fmt.Sprintf("CREATE KEYSPACE %s WITH replication = %s", tc.name, tc.replicationStmt)); err != nil {
+			ksStmt := fmt.Sprintf("CREATE KEYSPACE %s WITH replication = %s", tc.name, tc.replicationStmt)
+			if tablets := os.Getenv("TABLETS"); tablets == "enabled" && tc.replication == scyllaclient.SimpleStrategy {
+				ksStmt += " AND tablets = {'enabled': false}"
+			}
```

Comment:
In general, I don't really like relying on checking env variables in tests.
We do it in many places for convenience and out of laziness, because we don't really have vnode or tablet specific tests - we just run all the tests twice, once with tablets as default replication type, once with vnodes, but this is dirty and results in rerunning many tests for no reason at all (as they don't depend on tablet vs vnode).
If possible, it should be replaced with checking scylla version / default replication type.